addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -72,8 +72,8 @@
             </div>
         </t>
     </xpath>
-    <xpath expr="//div[@id='snippet_options']" position="inside">
-        <t t-call="mass_mailing.snippet_options"/>
+    <xpath expr="//div[@id='snippet_options']/t" position="attributes">
+        <attribute name="t-call">mass_mailing.snippet_options</attribute>
     </xpath>
```

Comment:
Make that an independant fix. Same branch, just different commit before the main one.